src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  function identifyPartner() {
+    if (isMarried) {
+      if (sex === 'm') {
+        return `${partnerName} is my wife`;
+      }
+
+      return `${partnerName} is my husband`;
+    }
+
+    return 'I am not married';
+  }
```

Comment:
do you really want to create new function on each rerender? move it outside of your component